src/ansys/hps/client/rms/models.py

```diff
@@ -34,683 +34,836 @@
+        title='Cool Down Period',
     )
 
 
 class ScalerRegistration(BaseModel):
-    id: Optional[str] = Field(None, description="Unique ID for this worker.", title="Id")
+    id: Optional[str] = Field(None, description='Unique ID for this worker.', title='Id')
     name: Optional[str] = Field(
-        None, description="User-defined name for this worker.", title="Name"
+        None, description='User-defined name for this worker.', title='Name'
     )
     last_modified: Optional[datetime] = Field(
         None,
-        description="Date and time when the registration was last modified.",
-        title="Last Modified",
+        description='Date and time when the registration was last modified.',
+        title='Last Modified',
     )
     host_id: Optional[str] = Field(
-        None, description="Static, hardware and configuration-based UUID.", title="Host Id"
+        None, description='Static, hardware and configuration-based UUID.', title='Host Id'
     )
     host_name: Optional[str] = Field(
-        None, description="Name of the host that the worker is running on.", title="Host Name"
+        None, description='Name of the host that the worker is running on.', title='Host Name'
     )
     username: Optional[str] = Field(
-        None, description="Username that the worker authenticated with.", title="Username"
+        None, description='Username that the worker authenticated with.', title='Username'
     )
     platform: Optional[str] = Field(
-        None, description="OS that the evaluator is running on.", title="Platform"
+        None, description='OS that the evaluator is running on.', title='Platform'
     )
     build_info: Optional[Dict[str, Any]] = Field(
-        {}, description="Detailed build information.", title="Build Info"
+        {}, description='Detailed build information.', title='Build Info'
     )
     config_modified: Optional[datetime] = Field(
         None,
         description="Date and time of the configuration's last modification.",
-        title="Config Modified",
+        title='Config Modified',
     )
 
 
 class ScalersCountResponse(BaseModel):
-    num_scalers: Optional[int] = Field(0, title="Num Scalers")
+    num_scalers: Optional[int] = Field(0, title='Num Scalers')
 
 
 class ScalersRequest(BaseModel):
-    scalers: List[ScalerRegistration] = Field(..., description="Scaler details", title="Scalers")
+    scalers: List[ScalerRegistration] = Field(..., description='Scaler details', title='Scalers')
 
 
 class ScalersResponse(BaseModel):
-    scalers: List[ScalerRegistration] = Field(..., description="Scaler details", title="Scalers")
+    scalers: List[ScalerRegistration] = Field(..., description='Scaler details', title='Scalers')
 
 
 class ServiceUserProcessRunner(BaseModel):
-    plugin_name: Literal["service_user_module"] = Field(..., title="Plugin Name")
+    plugin_name: Literal['service_user_module'] = Field(..., title='Plugin Name')
```

Comment:
Just because your content is different than most PyAnsys libraries, I'm not editing the ``title``. I assume it's different than the ``Field``. Sorry if my edits aren't appropriate. For Booleans, we always start the description with "Whether the X." We also end the brief description with a period.